### ISSUE TITLE
auto match doesnt take current scope into account

### ISSUE THREAD

**Issue by JonathanStarup** [flix/flix#9501]
When I'm trying to autocomplete a match on `A.B.C.D` in namespace `A.B.C.D`, the autocomplete still writes `A.B.C.D.Tag` instead of just tag. And in the general case, if I'm in namespace `A.B`, then I would want it to write `C.D.Tag` so it seems to be about matching the current namespace to the prefix of the enum qualifier

**Comment by stormckey** [comment]
You are right. To fix this problem we need the current namespace, which seems inaccessible from the semantic completions🤔. But where can we get this information?

**Comment by JonathanStarup** [comment]
Aah, right, hmm. maybe you can find the function you are in, and then look at its symbol and check its namespace?

**Comment by magnus-madsen** [comment]
Its not easy at the moment.
